br/pkg/streamhelper/advancer.go

```diff
@@ -448,8 +470,10 @@ func (c *CheckpointAdvancer) SpawnSubscriptionHandler(ctx context.Context) {
 	defer c.subscriberMu.Unlock()
 	c.subscriber = NewSubscriber(c.env, c.env, WithMasterContext(ctx))
 	es := c.subscriber.Events()
+	log.Info("Subscription handler spawned.", zap.String("category", "log backup subscription manager"))
 
 	go func() {
+		defer utils.CatchAndLogPanic()
```

Comment:
if it catch the panic here. but this goroutine has no longer exists anymore.
do we need generate this for loop again?

Reply:
If a random panic happens, the caller may or may not require retry this loop. I think it will be better to don't regenerate this loop and make the caller to decide whether to spawn another loop. 

Perhaps we can add some mark about whether the goroutine is still running later. Anyhow that is out of this PR's scope, which focus on restricting the panic in the scope of log backup advancer.